input/postgres/server_stats.go

```diff
@@ -0,0 +1,53 @@
+package postgres
+
+import (
+	"database/sql"
+
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+const transactionIdSQLPg13 string = `
+SELECT
+	pg_current_xact_id(),
+	next_multixact_id
+FROM pg_catalog.pg_control_checkpoint()
+`
+
+const transactionIdSQLPg96 string = `
+SELECT
+	txid_current(),
+	next_multixact_id
+FROM pg_catalog.pg_control_checkpoint()
+`
+
+const transactionIdSQLDefault string = `
+SELECT
+	txid_current(),
+	1
+`
+
+func GetServerStats(logger *util.Logger, db *sql.DB, postgresVersion state.PostgresVersion, systemType string) (state.PostgresServerStats, error) {
+	var stats state.PostgresServerStats
+	var transactionIdSQL string
+
+	// Only collect xact ID related stats with non-replicas
+	if isReplica, err := getIsReplica(db); err == nil && !isReplica {
```

Comment:
Not a big deal, but we could save a query if we get this once and remember it for both server stats and replication stats - similar to how we handle Postgres versions (they are fetched at the beginning of the input function).

Reply:
Very good point. Currently `GetIsReplica` actually returns more than "if it's replica", therefore I had to introduce `getIsReplica`.
I think we ideally want to save this query indeed (and actually `GetIsAwsAurora` too, which is used in `GetIsReplica`), but it comes with "naming is hard" problem so I'm gonna just leave it as is this time.